src/components/resumes/Details.jsx

```diff
@@ -1,20 +1,37 @@
 import Button from "@material-ui/core/Button";
 import Chip from "@material-ui/core/Chip";
+import DirectionsIcon from "@material-ui/icons/Directions";
+import Divider from "@material-ui/core/Divider";
 import Grid from "@material-ui/core/Grid";
+import IconButton from "@material-ui/core/IconButton";
+import InputBase from "@material-ui/core/InputBase";
 import Loading from "./Loading";
+import MenuIcon from "@material-ui/icons/Menu";
 import NotFound from "./NotFound";
 import Paper from "@material-ui/core/Paper";
+import SearchIcon from "@material-ui/icons/Search";
 import TextField from "@material-ui/core/TextField";
 import avatar from "../../assets/img_avatar.jpg";
```

Comment:
you can group the imports of materialUI something like that:
import {
  Button,
  Container,
  Divider,
  Grid,
  Typography,
} from "@material-ui/core";

Reply:
Done